sklearn/tests/test_multiclass.py

```diff
@@ -239,7 +239,7 @@ def test_ovr_multiclass():
         clf = OneVsRestClassifier(base_clf).fit(X, y)
         assert_equal(set(clf.classes_), classes)
         y_pred = clf.predict(np.array([[0, 0, 4]]))[0]
-        assert_equal(set(y_pred), set("eggs"))
+        assert_equal(set(y_pred), {"eggs"})
```

Comment:
I have no idea how that passed -- it shouldn't have,
```py
>>> set("eggs")
{'g', 'e', 's'}
>>> set(['eggs'])
{'eggs'}
```

Curisous to see this this fix will fail..